lib/Basics/files.cpp

```diff
@@ -1989,6 +1977,20 @@ static int mkDTemp(char* s, size_t bufferSize) {
     tmp = fromWString(ws);
     memcpy(s, tmp.data(), bufferSize);  // copy back into parameter
     rc = TRI_MKDIR(s, 0700);
+    if (rc != 0) {
+      rc = errno;
+      if (rc == ENOENT) {
+        // for some reason we should create the upper directory too?
+        std::string error;
+        long systemError;
+        rc = TRI_CreateRecursiveDirectory(s, systemError, error);
+        if (rc != 0) {
+          LOG_TOPIC("6656f", ERR, arangodb::Logger::FIXME)
+            << "Unable to create temporary directory " << rc << "error: " << error;
```

Comment:
this error message will look strange.
```suggestion
            << "Unable to create temporary directory: " << error;
```